packages/core-plugin-api/api-report.md

```diff
@@ -503,7 +503,7 @@ export const googleAuthApiRef: ApiRef<
 
 // @public
 export type IconComponent = ComponentType<{
-  fontSize?: 'default' | 'small' | 'large';
+  fontSize?: 'inherit' | 'medium' | 'large' | 'small';
```

Comment:
this would ignore `'default'` fontSize - for context: it was deprecated/replaced by 'medium' : https://mui.com/material-ui/migration/v5-component-changes/#icon

Reply:
hmm. will check .